networking/hardware_networks/configuring-sriov-device.adoc

```diff
@@ -9,7 +9,8 @@ You can configure a Single Root I/O Virtualization (SR-IOV) device in your clust
 
 include::modules/nw-sriov-device-discovery.adoc[leveloffset=+1]
 include::modules/nw-sriov-configuring-device.adoc[leveloffset=+1]
+include::modules/nw-sriov-nic-partitioning.adoc[leveloffset=+1]
```

Comment:
@jboxman To add it to webscale only, you'll just need to use these conditionals around the include:
ifdef::openshift-webscale[]
include::modules/nw-sriov-nic-partitioning.adoc[leveloffset=+1]
endif::openshift-webscale[]

Thanks for getting this added in!